### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuliiaSaliuta**

### CODE REVIEW

**Comment by Naz-iv** [comment]
well done

**Review by Naz-iv — COMMENTED** [review]
all right

**Review by Naz-iv — APPROVED** [review]
alright

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+        "path": file_path,
+        "status": "failed" if errors else "passed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        format_single_linter_file(linter, linter_report[linter])
```
Comment (fsocie7y) [inline app/main.py]: Please, use more descriptive variable names.

**Review by fsocie7y — CHANGES_REQUESTED** [review]

**Review by fsocie7y — APPROVED** [review]
well done ^_^

**Pull request closed without merging**